packages/demand-bloc-trip-create/src/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const defaultTripCreateOptions = {
+  minLengthToSearch: 3,
+  autocompleteDebounceTime: 500,
+  autocompleteLocationRadius: 1100000,
```

Comment:
Currently we have 2500000  radius. Is it something was changed?

Reply:
Thanks for noticed it. I assume we used to use it before changed the value on instances directly